libvncclient/corre.c

```diff
@@ -46,7 +46,10 @@ HandleCoRREBPP (rfbClient* client, int rx, int ry, int rw, int rh)
     if (!ReadFromRFBServer(client, (char *)&pix, sizeof(pix)))
 	return FALSE;
 
-    FillRectangle(client, rx, ry, rw, rh, pix);
+    if(client->GotFillRect != NULL)
```

Comment:
I agree. It would be much nicer to initialize the `GotFillRect` attribute to `FillRectangle` in `rfbGetClient()`.